aapenimservice/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/aapenimservice/AapenImService.kt

```diff
@@ -53,8 +52,7 @@ class AapenImService(
         Key.VIRKSOMHET,
         Key.ARBEIDSTAKER_INFORMASJON,
         Key.ARBEIDSGIVER_INFORMASJON,
-        Key.AAPEN_INNTEKTMELDING,
-        Key.ER_DUPLIKAT_IM
```

Comment:
Tar duplikatkontroll på sikt. Det er annerledes enn for forespurt IM, siden vi for åpen vil ha to innganger (helt ny og sendt inn).